turbomoleio/output/parser.py

```diff
@@ -2125,6 +2184,75 @@ def mp2_results(self):
 
         return dict(energy=energy)
 
+    @lazy_property
+    def periodicity_data(self):
+        """
+        Information about periodicity.
+
+        Returns:
+            dict with "periodicity", "tm_lattice_params", "shortest_interatomic_distance",
+                "direct_space_vectors", "reciprocal_space_vectors".
+        """
+        r = r"Periodic system found\: PBC structure information.*"
```

Comment:
does the output look the same if the structure is defined with `$lattice`?

Reply:
Seems like it (see comment above on the PeriodicityData object)